shared/transforms/shared_shorthand2xccdf.xslt

```diff
@@ -109,6 +109,22 @@
             </xsl:otherwise>
           </xsl:choose>
           <xsl:choose>
+            <xsl:when test="contains(requires/@prodtype, $prod_type) or requires/@prodtype = 'all'">
+              <xsl:apply-templates select="requires"/>
+            </xsl:when>
+            <xsl:otherwise>
+              <xsl:apply-templates select="requires[not(requires/@prodtype)]"/>
```

Comment:
I don't understand this condition. Can't it be simplified and added to the first if?